example/article/tests/test_comments.py

```diff
@@ -5,7 +5,10 @@
 
 from article.tests import factories
 from django.contrib.auth.models import User
-from django.core.urlresolvers import reverse
+try:
+    from django.urls import reverse
+except ImportError:  # Django<2.0
+    from django.core.urlresolvers import reverse
```

Comment:
Can this be moved below the import statements? (since it's code)

Reply:
Done.